os/include/tinyara/sched.h

```diff
@@ -617,7 +620,7 @@ struct tcb_s {
 	uint32_t uspace;		/* User space object for app binary */
 
 #ifdef CONFIG_ARMV7M_MPU
-	uint32_t mpu_regs[MPU_REGS];
+	uint32_t mpu_regs[3 * MPU_NUM_REGIONS];
```

Comment:
Could you remain description why we need 3 * MPU_NUM_REGIONS?

Reply:
Ok.